### PR TITLE
update examples

### PR DESCRIPTION
**Pull request by kimbo**
- added some docs about the new `<doh-script>` tag
- moved examples directory around
- put example usage of `Resolver()` in main README

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -21,9 +21,10 @@ The purpose of dohjs is described well in the Internet standard document for DNS
 
 - Fully compliant DNS over HTTPS client implementation
 - Supports GET and POST wireformat queries
-- Works great in your browser
-- [Web interface](https://dohjs.org) to try dohjs
+- Custom `<doh-script>` tag to fetch scripts using DoH resolver of your choice (see [examples/doh-script](examples/doh-script) for example usage)
 - Command line DNS over HTTPS lookup tool
+- [Web interface](https://dohjs.org) to try dohjs
+- CORS proxy availabel on [https://dohjs.org](https://dohjs.org) to get past CORS errors associated with DoH ([CORS proxy source code here](https://github.com/byu-imaal/dohjs/blob/gh-pages/cors_proxy.js))
```
Comment (jacobgb24) [inline README.md]: typo

File: README.md [hunk 2]
```diff
@@ -42,9 +43,7 @@ npm install -g dohjs
 
 # Quickstart
 
-To get started hacking with dohjs, checkout our [examples](examples).
-
-Be sure to include doh.js in your HTML file. You can include it from [jsdelivr](https://www.jsdelivr.com/) or your local installation.
+First, be sure to include doh.js in your HTML file. You can include it from [jsdelivr](https://www.jsdelivr.com/) or your local installation.
 Make sure you put it before your other `<script>` tags.
```
Comment (jacobgb24) [inline README.md]: Does it need to be before? It's in the body for dohjs.org

File: examples/README.md [hunk 3]
```diff
@@ -0,0 +1,6 @@
+# DoHjs examples
+
+Directory | Description
+---|---
+[examples/basic](examples/basic) | Basic examples of using dohjs, including the `Resolver` class, and other helpful functions.
+[examples/doh-script](examples/doh-script) | This shows the use of the `<doh-script>` tag, which is basically like a `<script>` tag, only load scripts without using the user's system DNS resolver.
```
Comment (jacobgb24) [inline examples/README.md]: typo: only **it loads** scripts...

File: README.md (hunk 2 above)
Comment (kimbo) [reply README.md]: Mmm I guess it depends on how you're using it. See https://stackoverflow.com/questions/8996852/load-and-execute-order-of-scripts. I think I'll remove it for now.

**Pull request merged**